corehq/apps/reports/views.py

```diff
@@ -230,6 +232,11 @@ def good_configs(self):
         all_configs = ReportConfig.by_domain_and_owner(self.domain, self.request.couch_user._id)
         good_configs = []
         for config in all_configs:
+            try:
```

Comment:
Could we use `stale` param in `by_domain_and_owner` for this instead?

Reply:
I think you need to pass `stale=False`

Reply:
With this change, confirming that issue is not seen. Thanks.

https://github.com/dimagi/commcare-hq/commit/28d43edbce9b6c15832c087ae45dda9e569d3f3c